modules/mono/.editorconfig

```diff
@@ -14,6 +14,8 @@ max_line_length = 120
 csharp_indent_case_contents_when_block = false
 
 [*.cs]
+# IDE0005: Remove unnecessary using directives
+dotnet_diagnostic.IDE0005.severity = suggestion
```

Comment:
This change seems unrelated.

Reply:
Oops, didn't see I staged this in the commit. I'll remove it.